src/scripts/modules/media/body/body.coffee

```diff
@@ -85,8 +76,24 @@ define (require) ->
       else
         $els.hide()
 
+    goToPage: (pageNumber, href) ->
+      @model.setPage(pageNumber)
+      router.navigate href, {trigger: false}, => @parent.parent.parent.trackAnalytics()
+
+    getCoach: ->
+      moduleUUID = @model.getUuid()?.split('?')[0]
+      settings.conceptCoach?.uuids?[moduleUUID]
```

Comment:
`settings.conceptCoach.uuids` is an object, so no.  Also, looking into the [`getUuid` function](https://github.com/Connexions/webview/blob/master/src/scripts/models/contents/node.coffee#L191), it will only return strings, so `moduleUUID` would be an empty string in the most "falsiest" case.